Jetcaster/mobile/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -348,16 +343,16 @@ private fun HomeScreenReady(
                         },
                         showBackButton = navigator.isMainPaneHidden(),
                     )
-                },
-                mainPane = {
-                    HomeScreen(
-                        homeState = homeState,
-                        modifier = Modifier.fillMaxSize()
-                    )
-                },
-                modifier = Modifier.fillMaxSize()
-            )
-        }
+                }
+            },
+            mainPane = {
```

Comment:
nit: Since you're refactoring this code, it would make more sense to move the `mainPane` parameter before `sortingPane`.

Reply:
Makes sense, re-ordered in https://github.com/android/compose-samples/pull/1510/commits/69ef8ced4ddad92635d7168a54173e269f79c730